Druid/DRBasicCogInterpreterObjectMemoryPrimitives.class.st

```diff
@@ -0,0 +1,25 @@
+Class {
+	#name : #DRBasicCogInterpreterObjectMemoryPrimitives,
+	#superclass : #CogVMSimulatorLSB,
```

Comment:
Is this class used?

Reply:
There was an autocompletion mistake I did. Now it's referenced in #interpreterClassObjectMemoryPrimitives